node/network/statement-distribution/src/vstaging/mod.rs

```diff
@@ -419,6 +419,7 @@ pub(crate) async fn handle_active_leaves_update<Context>(
 		// New leaf: fetch info from runtime API and initialize
 		// `per_relay_parent`.
 
+		// TODO [now]: Didn't we already request this in `lib.rs`?
```

Comment:
Not quite. We requested it for the activated leaf, not all leaves in the implicit view. Probably lib.rs should just avoid this check and delegate it to within the modules. All yours!